src/gui/guiHyperText.h

```diff
@@ -177,6 +179,8 @@ class TextDrawer
 	inline s32 getHeight() { return m_height; };
 	void draw(const core::rect<s32> &clip_rect,
 			const core::position2d<s32> &dest_offset);
+	void drawBackgroundImage(video::IVideoDriver *driver, const ParsedText &m_text, const core::rect<s32> &clip_rect);
+	ParsedText &getText() { return m_text; }
```

Comment:
i don't like this very dangerous pattern, it's better to have a copy and a setting i think. ok it's optimized memory side, but having full access to a member ref sounds bad